Change figaro to rbenv-vars in "manual installation" documentation (#8575)
decidim/decidim: docs/modules/install/pages/manual.adoc

We recommend to have at least some basic proficiency in Ruby on Rails (a good starting point is http://guides.rubyonrails.org/getting_started.html[Getting Started with Ruby on Rails]) and have some knowledge on how gems work.

> In this guide, we'll see how to install rbenv, PostgreSQL, Node.js and, Decidim, and how to configure everything together for a (and this is important) a development environment.

Reviewer: ```suggestion
In this guide, we'll see how to install rbenv, PostgreSQL, Node.js and, Decidim, and how to configure everything together for a development environment. Mind that if you want to make a production deployment with real users this guide isn't enough, you should configure a web server (like nginx), backups, monitoring, etc. This is out of the scope of this guide, but you can follow the https://platoniq.github.io/decidim-install/[Platoniq guide].
```

Modify your secrets (see `config/database.yml`). For this you can use https://github.com/laserlemon/figaro[figaro], https://github.com/bkeepers/dotenv[dotenv] or https://github.com/rbenv/rbenv-vars[rbenv-vars]. You should always be careful of not uploading your plain secrets on git or your version control system. You can also upload the encrypted secrets, using the sekrets gem or if you're on Ruby on Rails greater than 5.1 you can do it natively.

For a development environment, and if you are using rbenv, we strongly recommend you to use the https://github.com/rbenv/rbenv-vars[rbenv-vars] to facilitate the edition of ENV vars.

First you'll need to install the rbenv-vars plugin:

[source,bash]
----
git clone https://github.com/rbenv/rbenv-vars.git "$(rbenv root)"/plugins/rbenv-vars
----

Then, in **any folder above your decidim generated application**, you need to create a file named `.rbenv-vars` and put your variables there:

[source,bash]
----
nano .rbenv-vars
----

> [source,env]

Reviewer: I'm not sure if this will be formatted correctly by asciidoc/antora, but we'll try it in a couple of minutes...
Also, maybe it'd be better to change it to the multiline string to file in bash (like `cat > EOF << .rbenv-vars` and so on), as if we're giving instruction with `nano` then we should also add the instructions to saving and quitting (Ctrl+X, Y, enter, etc) but for the moment I just want to merge this and as I have a couple other minors changes to docs in my TODO list, I'll tackle that one in another PR 😄

Modify your secrets (see `config/database.yml`). For this you can use https://github.com/laserlemon/figaro[figaro], https://github.com/bkeepers/dotenv[dotenv] or https://github.com/rbenv/rbenv-vars[rbenv-vars]. You should always be careful of not uploading your plain secrets on git or your version control system. You can also upload the encrypted secrets, using the sekrets gem or if you're on Ruby on Rails greater than 5.1 you can do it natively.

For a development environment, and if you are using rbenv, we strongly recommend you to use the https://github.com/rbenv/rbenv-vars[rbenv-vars] to facilitate the edition of ENV vars.

> Using Rbenv-vars, you only need to perform one step (once) to add the automated ENV vars functionality to rbenv:

Reviewer: ```suggestion
First you'll need to install the rbenv-vars plugin:
```

[source,bash]
----
git clone https://github.com/rbenv/rbenv-vars.git "$(rbenv root)"/plugins/rbenv-vars
----

> Then, you just need to edit, in any folder above your decidim copy, a file named `.rbenv-vars` and put your variables there:

Reviewer: ```suggestion
Then, in **any folder above your decidim generated application**, you need to create a file named `.rbenv-vars` and put your variables there:
```

[source,bash]
----
nano .rbenv-vars
----

[source,env]
----
DATABASE_HOST=localhost
DATABASE_USERNAME=decidim_app
DATABASE_PASSWORD=thepassword
----

> 

Reviewer: ```suggestion

Be careful where you put the `.rbenv-vars` file, as if you put it in the same folder of your decidim generated application, and if you use a version control system (like `git`, which we strongly recommend), then you should ignore this file (ie with the `.gitignore` file)
```

> === 5.1 Alternatives with helper gems

Reviewer: Let's remove this block until the "Finally, save it all to git:" line
